sdk/spring/azure-spring-boot-starter-metrics/README.md

```diff
@@ -49,17 +54,24 @@ To solve this issue, you need to add the dependency below into your classpath:
    <version>2.3.0</version>
 </dependency>
 ```
+### Enable client logging
+Azure SDKs for Java offer a consistent logging story to help aid in troubleshooting application errors and expedite their resolution. The logs produced will capture the flow of an application before reaching the terminal state to help locate the root issue. View the [logging][logging] wiki for guidance about enabling logging.
 
 ## Next steps
 ### Allow telemetry
 Microsoft would like to collect data about how users use this Spring boot starter. Microsoft uses this information to improve our tooling experience. Participation is voluntary. If you don't want to participate, just simply disable it by setting below configuration in `application.properties`.
 ```
-azure.keyvault.allow.telemetry=false
+azure.servicebus.allow-telemetry=false
```

Comment:
Is this the wrong key? Why do we keep the telemetry section here while removing it from others?